<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcousticBrainzOptionsPage</class>
 <widget class="QWidget" name="AcousticBrainzOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>408</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="acoustic_features">
     <property name="title">
      <string>AcousticBrainz features extraction</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="disable_acoustic_features">
        <property name="text">
         <string>Do not use acoustic feature extraction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_acoustic_features">
        <property name="text">
         <string>Use acoustic feature extraction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="acousticbrainz_settings">
     <property name="title">
      <string>AcousticBrainz Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>AcousticBrainz/Essentia feature extractor:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="acousticbrainz_extractor"/>
        </item>
        <item>
         <widget class="QPushButton" name="acousticbrainz_extractor_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="acousticbrainz_extractor_download">
          <property name="text">
           <string>Download...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="acousticbrainz_extractor_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
